Source/WebCore/page/Quirks.cpp

```diff
@@ -569,19 +569,6 @@ bool Quirks::shouldPreventDispatchOfTouchEvent(const AtomString& touchEventType,
 
 #endif
 
-#if ENABLE(IOS_TOUCH_EVENTS)
-// mail.yahoo.com rdar://59824469
-bool Quirks::shouldSynthesizeTouchEvents() const
-{
-    if (!needsQuirks())
-        return false;
-
-    if (!m_shouldSynthesizeTouchEventsQuirk)
-        m_shouldSynthesizeTouchEventsQuirk = isYahooMail(*protectedDocument());
```

Comment:
Too bad we can't get rid of the `isYahooMail` predicate entirely!

Reply:
fixed the commit message. 
Yeah. Maybe we will be able in the refactoring of Quirks.